### ISSUE TITLE
no intergenic peaks ?

### ISSUE THREAD

**Issue by gnilihzeux** [scottzijiezhang/MeRIPtools#5]
Dear author,
    It seems that MeRIPtools focuses peaks on regions with annotation.
    Is that so ?

**Comment by scottzijiezhang** [comment]
Yes it was originally designed to analyze mRNA data. I recently added functions specifically for pre-mRNA (without skipping introns).
